tools/cli/src/mentat_cli/command_parser.rs

```diff
@@ -129,6 +135,15 @@ pub fn command(s: &str) -> Result<Command, cli::Error> {
                         }
                         Ok(Command::Close)
                     });
+                    
+    let schema_parser = string(SCHEMA_COMMAND)
```

Comment:
I think this might be an unnecessary rabbit hole.

Take a look at https://docs.rs/combine/2.3.2/combine/fn.not_followed_by.html instead?